ports/libsimpleservo/api/Cargo.toml

```diff
@@ -10,7 +10,9 @@ publish = false
 libservo = { path = "../../../components/servo" }
 log = "0.4"
 servo-media = { git = "https://github.com/servo/media" }
-webxr-api = { git = "https://github.com/servo/webxr" }
+
+webxr-api = { git = "https://github.com/servo/webxr", features = ["ipc"] }
+webxr = { git = "https://github.com/servo/webxr" , optional = true}
```

Comment:
Hmm, bit odd that libsimpleservo needs to know about `webxr`, but I guess this is inevitable if the uwp port is using c_api rather than a Rust AOI.

Reply:
Yes, if the UWP port moves to libhlservo (probably a good idea in the long run) we can fix this